Test failure on Django 3.1a1

There's a few changes in Django 3.1 around date times and ISO parsing. 
https://docs.djangoproject.com/en/3.1/releases/3.1/

[We've got one failure here](https://travis-ci.org/github/carltongibson/django-filter/jobs/687053341#L286-L296): 

```
======================================================================
287FAIL [0.002s]: test_clean (tests.test_fields.IsoDateTimeRangeFieldTests)
288----------------------------------------------------------------------
289Traceback (most recent call last):
290  File "/home/travis/build/carltongibson/django-filter/.tox/py37-latest/lib/python3.7/site-packages/django/test/utils.py", line 381, in inner
291    return func(*args, **kwargs)
292  File "/home/travis/build/carltongibson/django-filter/tests/test_fields.py", line 107, in test_clean
293    datetime(2015, 1, 10, 7, 45, 2, 345000)))
294AssertionError: slice[26 chars], 1, 10, 30, 1, 123000, tzinfo=datetime.timezo[167 chars]None) != slice[26 chars], 1, 9, 30, 1, 123000), datetime.datetime(2015[28 chars]None)
295
296----------------------------------------------------------------------
```

--- a/tests/test_fields.py
+++ b/tests/test_fields.py
@@ -97,14 +97,15 @@ class IsoDateTimeRangeFieldTests(TestCase):
         f = IsoDateTimeRangeField()
         self.assertEqual(len(f.fields), 2)
 
-    @override_settings(USE_TZ=False)
     def test_clean(self):
         w = RangeWidget()
         f = IsoDateTimeRangeField(widget=w)
-        self.assertEqual(
-            f.clean(['2015-01-01T10:30:01.123000+01:00', '2015-01-10T08:45:02.345000+01:00']),
-            slice(datetime(2015, 1, 1, 9, 30, 1, 123000),
-                  datetime(2015, 1, 10, 7, 45, 2, 345000)))
+        expected = slice(
+            datetime(2015, 1, 1, 9, 30, 1, 123000, tzinfo=timezone.utc),
+            datetime(2015, 1, 10, 7, 45, 2, 345000, tzinfo=timezone.utc)
+        )
+        actual = f.clean(['2015-01-01T10:30:01.123000+01:00', '2015-01-10T08:45:02.345000+01:00'])
+        self.assertEqual(expected, actual)
 
 
 class TimeRangeFieldTests(TestCase):

